python_modules/libraries/dagster-airflow/dagster_airflow/utils.py

```diff
@@ -11,23 +11,31 @@
     _check as check,
 )
 from dagster._core.definitions.utils import VALID_NAME_REGEX
+from packaging import version
+
+
+def is_airflow_2() -> bool:
```

Comment:
I'd prefer this function named `is_airflow_2_loaded_in_environment` or something. `is_airflow_2` is a function that sounds like it should have a parameter inspected. `is_airflow_2_loaded_in_environment` makes it clear it is just inspecting some magical global.